src/commands/README.md

```diff
@@ -1,13 +1,216 @@
+Command JSON files
+==================
+
 This directory contains JSON files, one for each command.
 
-Each JSON contains all the information about the command itself, but these JSON files are not to be used directly!
-Any third party who needs access to command information must get it from `COMMAND INFO` and `COMMAND DOCS`.
-The output can be extracted in a JSON format by using `redis-cli --json`, in the same manner as in `utils/generate-commands-json.py`.
+Each JSON file contains all the information about the command itself. It is the
+"single source of truth" (SSOT) for the command's metadata.
+
+These JSON files were originally not intended to be used directly, since they
+contain internals and some fields like "acl_categories" that are not the final
+ACL categories. (Valkey will apply some implicit rules to compute the final ACL
```

Comment:
Maybe we should make an issue about the implicit rules and reference the issue here so we have a paper trail?

Reply:
Sure, will do